test-various works when ran locally, and it seems like the error was that the gnu mirror kept timing out. Maybe it was a one-off network issue when running CI? 

```
#13 329.5 failed: Connection timed out.
#13 329.5 Connecting to gitweb.git.savannah.gnu.org (gitweb.git.savannah.gnu.org)|135.148.138.35|:443... failed: Connection timed out.
#13 329.5 Connecting to gitweb.git.savannah.gnu.org (gitweb.git.savannah.gnu.org)|38.102.85.135|:443... connected.
--- ERROR ---
#13 329.5 HTTP request sent, awaiting response... 502 Bad Gateway
#13 329.5 2025-07-01 21:42:40 ERROR 502: Bad Gateway.
#13 329.5
#13 329.5 make: *** [Makefile:79: sources/config.sub] Error 8
--- ERROR ---
#13 ERROR: process "/bin/sh -c bash musl-toolchain.sh x86_64 && rm -rf build" did not complete successfully: exit code: 1
```

--- a/src/bootstrap/src/core/sanity.rs
+++ b/src/bootstrap/src/core/sanity.rs
@@ -334,7 +334,8 @@ than building it.
                 || build.config.rust_std_features.contains("compiler-builtins-c"))
         {
             let cc_tool = build.cc_tool(*target);
-            if !cc_tool.is_like_clang() {
+            if !cc_tool.is_like_clang() && !cc_tool.path().ends_with("emcc") {
+                // emcc works as well
                 panic!(
                     "Clang is required to build C code for Wasm targets, got `{}` instead\n\
                     this is because compiler-builtins is configured to build C source. Either \

